cloud/types.go

```diff
@@ -277,6 +278,29 @@ type (
 		GitlabCICDMergeRequestApproved *bool `json:"gitlab_cicd_merge_request_approved,omitempty"` // CI_MERGE_REQUEST_APPROVED
 	}
 
+	// BitbucketMetadata holds the Bitbucket specific metadata.
+	BitbucketMetadata struct {
+		BitbucketPipelinesBuildNumber               string `json:"bitbucket_pipelines_build_number,omitempty"`
+		BitbucketPipelinesPipelineUUID              string `json:"bitbucket_pipelines_pipeline_uuid,omitempty"`
+		BitbucketPipelinesCommit                    string `json:"bitbucket_pipelines_commit,omitempty"`
+		BitbucketPipelinesWorkspace                 string `json:"bitbucket_pipelines_workspace,omitempty"`
+		BitbucketPipelinesRepoSlug                  string `json:"bitbucket_pipelines_repo_slug,omitempty"`
+		BitbucketPipelinesRepoUUID                  string `json:"bitbucket_pipelines_repo_uuid,omitempty"`
+		BitbucketPipelinesRepoFullName              string `json:"bitbucket_pipelines_repo_full_name,omitempty"`
+		BitbucketPipelinesBranch                    string `json:"bitbucket_pipelines_branch,omitempty"`
+		BitbucketPipelinesDestinationBranch         string `json:"bitbucket_pipelines_destination_branch,omitempty"`
+		BitbucketPipelinesTag                       string `json:"bitbucket_pipelines_tag,omitempty"` // only available in tag events.
+		BitbucketPipelinesStepTriggererUUID         string `json:"bitbucket_pipelines_step_triggerer_uuid,omitempty"`
+		BitbucketPipelinesParallelStep              string `json:"bitbucket_pipelines_parallel_step,omitempty"`
+		BitbucketPipelinesParallelStepCount         string `json:"bitbucket_pipelines_parallel_step_count,omitempty"`
+		BitbucketPipelinesPRID                      string `json:"bitbucket_pipelines_pr_id,omitempty"` // only available in PR events.
+		BitbucketPipelinesStepUUID                  string `json:"bitbucket_pipelines_step_uuid,omitempty"`
```

Comment:
Miss the `bitbucket_pipelines_step_name` to know the step where Terramate CLI was run

Reply:
There's no step name documented here: https://support.atlassian.com/bitbucket-cloud/docs/variables-and-secrets/

I have a pipeline with a `env` command here: https://bitbucket.org/terramate/terramate/pipelines/results/55
it only have the `BITBUCKET_STEP_UUID`

Reply:
By calling step API endpoint

Reply:
Using Step UUID in first iteration.